source/production/F0.Minesweeper.Logic/LocationShuffler/DefaultLocationShuffler.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using F0.Minesweeper.Logic.Abstractions;
+
+namespace F0.Minesweeper.Logic.LocationShuffler
+{
+	internal class DefaultLocationShuffler : ILocationShuffler
+	{
+		IReadOnlyCollection<Location> ILocationShuffler.ShuffleAndTake(IEnumerable<Location> allLocations, int count)
```

Comment:
should we throw, if `count` is _greater than_ the available locations?

Reply:
Yes, good idea, thanks!